packages/fiber/src/core/store.ts

```diff
@@ -70,6 +70,7 @@ export type RootState = {
   controls: THREE.EventDispatcher | null
   raycaster: Raycaster
   mouse: THREE.Vector2
+  dpr: Dpr
```

Comment:
why do we put dpr in here? it's a complex type, number or [number, number]

Reply:
I put that there to compare with the last uncalculated DPR prop in [web/index](https://github.com/pmndrs/react-three-fiber/blob/fix/setdpr/packages/fiber/src/web/index.tsx#L71) but I suppose I can compare the calculated values instead.

Reply:
yes i think makes sense to compare against the lastProps calculated value because [1, 2] !== [1, 2] in javascript, both are distinct arrays.